### DISCUSSION TITLE
Chromium version 116 error print

### DISCUSSION THREAD

**Discussion by khanht9x** [gotenberg/gotenberg#989] [Q&A]
I am using gotenberg 8 but with chromium version 116 I am having an error when printing PDF from URL. My text is overwritten. I tried Ctrl P in the browser but it is also overwritten. I still want to use gotenberg 8 but want to install Chromium v99, is that possible? Please help me
![Uploading Screenshot 2024-10-01 at 9.34.31 PM.png…]()

**Comment by khanht9x** [comment]
<img width="622" alt="Screenshot 2024-10-01 at 9 34 31 PM" src="https://github.com/user-attachments/assets/3c6dbcc4-06f7-4de0-90f6-b90b6fbb5db4">

**Comment by khanht9x** [comment]
I tried chromium v99 and didn't see the text overlap problem

**Comment by gulien** [comment]
Hello @khanht9x,

Are you running Gotenberg on a `armhf` architecture?

if you want to build a custom variant of Gotenberg, you may try: https://github.com/gotenberg/gotenberg/blob/1252ea076bc53127c29d85188a015e51b022572c/build/Dockerfile#L134

But I have not enough time to help you debug.

**Comment by khanht9x** [comment]
> Hello @khanht9x,
> 
> Are you running Gotenberg on a `armhf` architecture?
> 
> if you want to build a custom variant of Gotenberg, you may try:
> 
> https://github.com/gotenberg/gotenberg/blob/1252ea076bc53127c29d85188a015e51b022572c/build/Dockerfile#L134
> 
> But I have not enough time to help you debug.

I tried running puppeteer with chromium 99 and it work. I also tried using chrome browser with chromium 99 and it work.

I really like gotenberg 8 and want to use it with chromium 99. I'm running gotenberg using k8s and running Docker image

I don't know how to customize to run gotenberg 8 with chromium version 99

**Comment by gulien** [comment]
> I don't know how to customize to run gotenberg 8 with chromium version 99

Dunno if it's possible, but you may try to:

1. Fork this repository.
2. Update the `gotenberg/build/Dockerfile` by setting a particular version like I did for the `armhf` architecture.
3. Run the `make build` command.
